src/binary/binary2/traits.cpp

```diff
@@ -0,0 +1,216 @@
+namespace gdf {
+namespace cuda {
+
+const char* traits =
+R"***(
+#pragma once
+
+    struct IntegralSigned {};
```

Comment:
I had some issue with that. I expressed it in my daily scrum. It is supposed to be working, but it isn't. I included the libraries, however "common_type" and others aren't working, because of the error messages using nvrtc.

Reply:
Let's work with Kate Clark and Ben Barsdell on making Jitify better. Are you in touch with them?

Reply:
No. I'm not. I remember having issues related to "type_traits" header and with compiler option "--device-as-default-execution-space", which wasn't working.

Reply:
I will contact them.